frontend/packages/ceph-storage-plugin/src/components/create-storage-system/create-storage-system-steps/capacity-and-nodes-step/stretch-cluster.tsx

```diff
@@ -61,6 +61,7 @@ export const StretchCluster: React.FC<StretchClusterProps> = ({
         aria-label={t('ceph-storage-plugin~Enable arbiter')}
         id="stretch-cluster"
         isChecked={enableArbiter}
+        data-checked-state={enableArbiter}
```

Comment:
The other approach to adding the data attribute to every instance of PF's Checkbox component is to write a simple wrapper component that applies the data attribute.  With that said, it seems like we want, whenever possible, to use the base PF component. 

Adding a wrapper component would add more code changes immediately (as part of this bug), but would enforce consistency.   I leaned toward adding fewer code changes to address this bug.